tests/foreman/api/test_repository.py

```diff
@@ -1387,6 +1387,46 @@ def test_positive_sync_rh_app_stream(self):
         """
         pass
 
+    @pytest.mark.stubbed
+    @pytest.mark.tier1
+    def test_positive_bulk_cancel_sync(self):
+        """Bulk cancel 10+ repository syncs
+
+        :id: f9bb1c95-d60f-4c93-b32e-09d58ebce80e
+
+        :steps:
+            1. Add 10+ repos and sync all of them
+            2. Cancel all of the syncs
+            3. Check /var/log/foreman/production.log and /var/log/messages
+
+        :expectedresults: All the syncs stop successfully.
+
+        :CaseImportance: High
+
+        :CaseAutomation: NotAutomated
+        """
+        pass
+
+    @pytest.mark.stubbed
+    @pytest.mark.tier1
+    def test_positive_reboot_recover_sync(self):
+        """Reboot during repo sync and resume the sync when the Satellite is online
+
+        :id: 4f746e28-444c-4688-b92b-778a6e58d614
+
+        :steps:
+            1. Add a repo and start sync
+            2. Reboot the Satellite while sync is running
+            3. Check Foreman Tasks
+
+        :expectedresults: Repo sync resumes / can resume and finishes successfully
+
+        :CaseImportance: High
+
+        :CaseAutomation: NotAutomated
+        """
+        pass
```

Comment:
```suggestion
```
Is this required?

Reply:
it isn't. there are also a number of other instances of this in the pr

Reply:
Oh ok. I saw that for some other stubs and thought it was required. Removing now.